apps/rush/src/utilities/VersionManager.ts

```diff
@@ -189,7 +189,7 @@ export class VersionManager {
     clonedProject: IPackageJson,
     projectVersionChanged: boolean
   ): void {
-    if (!clonedProject.dependencies) {
+    if (!clonedProject.dependencies && !clonedProject.devDependencies) {
```

Comment:
>dependencies [](start = 23, length = 12)

What about optionalDependencies and peerDependencies?

Reply:
Those do not get version updated.